nginx/etc/nginx/nginx.conf

```diff
@@ -0,0 +1,136 @@
+user nginx;
+worker_processes 4;
+pid /run/nginx.pid;
+
+events {
+    worker_connections 768;
```

Comment:
I've been trying to standardize on 1024 here. Unclear what the best value is, but if things are the same it can at least give us a relative reference to another setup with the same value.